docs/migration/6_0/deleted-api.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "apiElementName": "SomeService",
+    "entryPoint": "@spartacus/feature/core",
+    "migrationComment": "Use class OtherService instead.  (note: this example won't appear in the migration material because the SomeService class does not exist)"
+  },
+  {
+    "apiElementName": "AwesomeClass",
+    "entryPoint": "@spartacus/feature/components",
+    "migrationComment": "This class is not needed anymore.  Things are done in this way instead.  (note: this example won't appear in the migration material because the AwesomeClass class does not exist)"
+  }
+]
```

Comment:
> not really a problem, but just letting you know. Old return types that were implicit are being put in the generated docs due to the fact that an explicit type was set to it now.

Indeed.  This is one of the limits of the tool.  If return types are different they will be listed amongst breaking changes.  To set correct expectations, at the top of the md doc file we announce that we have a list of breaking changes or _potentially_ breaking changes.